### ISSUE TITLE
Is it possible to running simulation of a small molecule in solvent rather than water  ( e.g  urea/anthracene in toluene ) ?

### ISSUE THREAD

**Issue by rijuaxt** [openmm/openmm#3222]
Is it possible to run simulation of a molecule in any solvent? if so , could you please let me know the process?
The conventional force field have templates for amino acid residues but not for small molecules. To sidestep this problem one may use GAFF or external server based file. However, what if one doesn't have any source of parametrizing more bigger molecule (Asphaltene) and different solvent (octanol. Ethanol etc.)  for simulation ?  Moreover, I am not so sure how a solute and solvent ratios could be adjusted ? For instance; if I wanted to run MD of Choline chloride with glycerol in 1:3 ratios in gas phase , what the process would be ?

If anybody contributes here significantly, that would be my  great help. 

@peastman @jchodera I am doing my MS thesis on this kind of things. I am out of my knowledge solving these issues i mentioned above.  I would be ever grateful to you all. 

Thanks

**Comment by peastman** [comment]
I believe that [OpenForceField Toolkit](https://github.com/openforcefield/openff-toolkit) should be able to parametrize any of those molecules.  I'm not sure what the best way of building the models is.  There may be tools that can build boxes of arbitrary solvents in user specified ratios.  That isn't something I've ever tried to do.  If you can't find anything, I can suggest a procedure for building them.  A specially designed tool would be better though.

**Comment by jchodera** [comment]
I'm not entirely sure whether the `openff-toolkit` supports `win`. Do you have a linux machine, or can you use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)?

**Comment by rijuaxt** [comment]
Hello,

In linux latest version,  i couldn’t even install Openmm itself. It shows
too many conflicts.

I tried changing anacondas from latest to old version, nothing worked.

Config: AMD 5950 64 bit ubuntu  20 + latest
RAM :64, GPU. 3090 24 GB, driver 371 latest, python 3.8 version



Suggest me for fresh installation Please.

**Comment by jchodera** [comment]
> In linux latest version,  i couldn’t even install Openmm itself. It shows too many conflicts

That's odd. I find my anaconda environment sometimes becomes inconsistent and I need to start from scratch, or even just install in a new conda environment. [Miniforge](https://github.com/conda-forge/miniforge) is a good way to do that.

If starting from a fresh miniforge doesn't work, could you post the error messages you're getting?

**Comment by rijuaxt** [comment]
@jchodera @peastman  so, you are suggesting ton install miniforge  and install e everything right?

So,  i Will uninstall anaconda, clean its everything and opt for miniforge for installing everything. 


Does miniforge cover most of the tools associated with openmm

**Comment by peastman** [comment]
Did you try simply creating a new environment with `conda create` like I suggested?  That ought to work.  If it doesn't, please post the exact error messages.

**Comment by rijuaxt** [comment]
@peastman I think Openmm have lot of problems , and a beginner should know most about it. I even saw a old cookbook tutorial that doesn't mitigate research types people need always. Unlike SCHRODINGER suite, MOE, or other software's, there are  enough flexibilities for atom typing by force field. While others have very limited problem. Could you please update your cookbook including some;  for instances; Protein-ligand simulation, Protein simulation with Surfactant, *MM/GBSA, MM/PBSA. making solvents. Examples are really helpful to modify and run simulation easily. You know a PhD or MS students hardly have time to go through everything at a time. Personally,  I have lost my focus doing this kind of technical work.  I have been working through openmm for a long time and found although it is intuitive to implement codes, resources are very limited. i think your 30 mins of work saves days and days of your followers

**Comment by jchodera** [comment]
@rijuaxt : We very much recognize OpenMM can do a much better job with materials to bring new users up to speed, but it's important to understand that OpenMM is not commercial software like the other packages you mentioned (Schrödinger Suite or MOE). If that level of commercial software support is critical to your work, we certainly recommend you pay for commercial software that will meet your needs.

OpenMM is open source software that has not been supported by a major federal grant since 2013. We have only recently secured a new source of federal funding, and this has let us bring on an additional part-time developer to help us improve documentation, tutorials, and cookbook recipes. Our focus is to make it easier for users new to OpenMM like yourself have an easier time getting up to speed quickly and tackling research problems of interest to the community, so your feedback is highly valuable!

If you have specific suggestions of what you would like to see, it helps us help you the most if you can file distinct issues with the necessary context around your request. Your issue https://github.com/openmm/openmm/issues/3228#issuecomment-907565707 is a great example of this: You provided a clearly-defined use case, some helpful references to understand this use case, and an example of tools that make it easy to tackle that use case in other codes. These kinds of suggestions can be incredibly helpful to us, though we do have to balance the needs of many different stakeholders in prioritizing work.

TL;DR: We recognize that the onboarding experience isn't as smooth as it could be, and we're working hard to fix that!

**Comment by peastman** [comment]
Closing this since the question was answered.
